lib/jazzy/doc_builder.rb

```diff
@@ -297,6 +297,13 @@ def self.document(source_module, doc_model, path_to_root)
         return document_index(source_module, path_to_root)
       end
 
+      tasks = render_tasks(source_module, doc_model.children)
+      tasks_list_children = tasks.map do |task|
+        if task[:name]
+          { name: task[:name], url: "#/"+task[:uid] }
+        end
+      end
```

Comment:
$ irb
    2.2.0 :001 > [1, 2, 3].flat_map { |x| x * 10 if x > 1 }
     => [nil, 20, 30] 

A case Ruby is actually stricter than Swift. Inform the media!